tests/storage/storage.go

```diff
@@ -259,8 +259,12 @@ var _ = SIGDescribe("Storage", func() {
 					libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 
 					if targetImagePath != testsuite.HostPathAlpine {
-						tests.DeleteAlpineWithNonQEMUPermissions()
-					}
+						nonQemuAlpinePath := testsuite.HostPathAlpine + "-nopriv"
+						args := []string{fmt.Sprintf(`rm -rf %s`, nonQemuAlpinePath)}
```

Comment:
Can you use quotes instead of backticks for the format string? Maybe inline `nonQemuAlpinePath` and add a comment above explainging that this should delete Alpine with non QEMU permissions instead?

Reply:
Can you inline `nonQemuAlpinePath`?